src/update.ts

```diff
@@ -1,7 +1,35 @@
+import chalk from 'chalk'
 import * as updateNotifier from 'update-notifier'
 
 import * as pkg from '../package.json'
 
+const notifier = updateNotifier({ pkg, updateCheckInterval: 1000 * 60 * 60 * 1 })
+
 export default function notify() {
-  updateNotifier({ pkg, updateCheckInterval: 1000 * 60 * 60 * 1 }).notify({ isGlobal: true, isYarnGlobal: true })
+  if (notifier.update && notifier.update.latest !== pkg.version) {
+    const oldVersion = notifier.update.current
+    const latestVersion = notifier.update.latest
+    const changelog = `https://github.com/vtex/toolbelt/releases/tag/v${latestVersion}`
```

Comment:
Unfortunately for now the release section on github doesn't contain details on the latest version :( - (we are working on a CI to automate this and improve our docs and releases)

I think for now just a url to the `master` chagelog would suffice, what do you think?

Reply:
Yup. We can do that!